Telegram/Resources/langs/lang.strings

```diff
@@ -1427,6 +1427,6 @@ Copyright (c) 2014-2017 John Preston, https://desktop.telegram.org
 "lng_mac_menu_add_contact" = "Add Contact";
 "lng_mac_menu_new_group" = "New Group";
 "lng_mac_menu_new_channel" = "New Channel";
-"lng_mac_menu_show" = "Show Telegram";
+"lng_mac_menu_show" = "Show {telegram}";
```

Comment:
`{telegram}` → `{appname}`?

Reply:
Mac-specific section contains the following strings:
```
"lng_mac_menu_hide_telegram" = "Hide {telegram}";
"lng_mac_menu_quit_telegram" = "Quit {telegram}";
"lng_mac_menu_about_telegram" = "About {telegram}";
```

So I decided to use `{telegram}` for `"lng_mac_menu_show"` too.

Reply:
I still suggest to replace if with `{appname}`. Guys, what would you say?

Reply:
It will not work, this line change change is bad because mac compilation should fail. Code calling this translation should be fixed as well, both with {telegram} constant or {appname} constant. I can't test build on Mac :)